pages/Example.md

```diff
@@ -4,19 +4,6 @@ title: Explained by example | Hydraecosystem.org
 permalink: /Example
 ---
 
-# Explained by example
```

Comment:
Did you move this or are you gonna add another explaination here?

The examples are using flock-demo as a reference. This para was there to give context.

Reply:
You would have not asked this question if you would have run the code of my branch. Which I don't think you did, despite many requests.